plugins/labormanager.cpp

```diff
@@ -2156,10 +2300,16 @@ class AutoLaborManager {
             {
                 skill_level = Units::getEffectiveSkill(d->dwarf, skill);
                 xp = Units::getExperience(d->dwarf, skill, false);
+
+                for (int pa = 0; pa < 6; pa++) 
```

Comment:
It looks like the whitespace here and on line 2307 is the issue. I'm not sure why the Travis script is reporting 2305/2308.